Dockerfile

```diff
@@ -12,6 +12,6 @@ RUN mix deps.get
 RUN mix compile
 RUN mix phx.digest
 RUN mix release --overwrite
-
+RUN _build/prod/rel/mind_the_gapp/bin/mind_the_gapp eval "MindTheGapp.Release.migrate"
```

Comment:
`RUN` only happens when you build the container. Do we want to migrate the database on build, or on startup? On startup is definitely more conventional.

For instance, if the database is not available, should that prevent you from building an image (perhaps for starting it later)?

Reply:
It's in the Dockerfile because `mix release` is called there and `_build/` is updated/generated in that command. 

If the migration is called outside the docker image, `_build/` may not exist, since it's generated with `mix release`.

Reply:
My question is more about **when** migrations should be run. When the Docker image is being built, or when a Docker container is started.